### ISSUE TITLE
Gosigar explodes when cgroup controllers are mounted in multiple FS locations

### ISSUE THREAD

**Issue by klakin-pivotal** [cloudfoundry/gosigar#54]
Gosigar currently panics when it looks for cgroup controllers that are mounted in multiple locations. It should not panic, as this is not illegal

For reference, the code in question is here: <https://github.com/cloudfoundry/gosigar/blob/746c62e150453ea6b14daae44d70777d513fa74b/sigar_linux.go>

Concerns:
* Is gosigar usually initialized once, and then used over a long period of time? If so, the thing that mounted the cgroup controller in the places after the first mount might unmount that controller, and then we'd go to read the data and would fail.
    * Could mitigate this by looking up the controller path every time we go to look up stats
    * Could also maybe mitigate this by stopping when we find the _first_ controller of interest in `/proc/self/mounts`... but that would only work if it were guaranteed that mounts higher up in the file happen before mounts later in the file.

**Comment by klakin-pivotal** [comment]
Additionally:

One thing we could also do to simplify this code and remove all the hand-rolled parsing of `/proc/self/mounts` is to shell out to `findmnt`

so
```
findmnt --types cgroup --options memory --uniq --output TARGET --noheadings --notruncate --canonicalize
```

and

```
findmnt --types cgroupv2 --options memory --uniq --output TARGET --noheadings --notruncate --canonicalize
```

should give us the path for the one or more mount points of the `memory` controller for either cgroupv1 or cgroupv2.

We could also see if the Golang standard library for listing mount points (if there even is one) is sane.

**Comment by jpalermo** [comment]
Fixed in version 1.3.3
